addons/project_mrp_account/models/mrp_production.py

```diff
@@ -0,0 +1,31 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _
+
+
+class MrpProduction(models.Model):
+    _inherit = 'mrp.production'
+
+    has_analytic_account = fields.Boolean(compute='_compute_has_analytic_account')
+
+    def _compute_has_analytic_account(self):
+        for production in self:
+            production.has_analytic_account = any(production.project_id._get_analytic_account_ids())
```

Comment:
We should recompute it if the project changed. 🙂 

Also, let's imagine we have the same project on all MOs in `self`, we will call `_get_analytic_account_ids` too much times since just one call was sufficient. 🙂 
We should create a dict with project as key and the result of `_get_analytic_accounts`.

```suggestion
    @api.depends('project_id')
    def _compute_has_analytic_account(self):
        has_analytic_account_per_project_id = {p.id: bool(p._get_analytic_account_ids()) for p in self.project_id}
        for production in self:
            production.has_analytic_account = has_analytic_account_per_project_id.get(production.project_id.id, False)
```